projects/lux/src/lib/input/input.component.scss

```diff
@@ -18,3 +23,23 @@
   background-color: $disabled-color;
   border-color: $disabled-border-color;
 }
+
+.icon {
+  display: inline-block;
+  background-size: 1rem 1rem;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-width: 1rem;
+  min-height: 1rem;
+  width: $icon-width;
+  z-index: 1;
+}
+.clickable {
+  cursor: pointer;
+}
+.icon-external {
+  @extend .icon;
+  background: $icon-external;
+  background-size: 1rem;
+  height: 100%;
```

Comment:
Add to center:
`background-position-y: 0.1rem;`

Reply:
Added: https://github.com/metadevpro/lux/pull/171/commits/10b6d084bddcf09add57b33d34fa4d457bff1ff1